src/transformers/models/bigbird_pegasus/modeling_bigbird_pegasus.py

```diff
@@ -1815,6 +1829,33 @@ def forward(
 
         if attention_mask is None:
             attention_mask = torch.ones(input_shape, device=hidden_states.device)
+        attention_mask = attention_mask.long()
```

Comment:
This makes `BigBirdPegasusModelTest.test_auto_padding` fail I think -> Also `block_sparse` attention in the decoder is really an edge case which I don't think makes too much sense to add for now. Do you think we could maybe skip it for now and maybe add at a later stage?